src/main/java/com/halotroop/arrchecker/ARRChecker.java

```diff
@@ -36,8 +45,8 @@ public void onInitialize() {
                 } else {
                     String[] validLicenses =
                             {
-                                    "gpl", "mit", "cc0", "apache", "unlicense", "mpl", // Short form names
-                                    "gnu public license", "mozilla public license", "creative commons" // Long form (incorrect, but check anyway)
+                                    "gpl", "lgpl", "mit", "cc0", "apache", "unlicense", "mpl", "wtfpl", // Short form names
+                                    "gnu lesser general public license", "mozilla public license", "creative commons", "cc by nc" // Long form (incorrect, but check anyway)
```

Comment:
You've added lesser, but haven't kept the normal GPL long-form.

I'm considering adding a separate warning about the FSF and GPL, by the way.